frontend/src/lib/components/TaxonomicFilter/cohortFilterUtils.ts

```diff
@@ -1,43 +0,0 @@
-import { BehavioralFilterKey } from 'scenes/cohorts/CohortFilters/types'
```

Comment:
this client side filter was only used in the FF UI, and now that I've moved this logic server-side I can delete it.